build/common/installer/scripts/tomlparser-prom-customconfig.rb

```diff
@@ -342,6 +346,13 @@ def populateSettingValuesFromConfigMap(parsedConfig)
                 file.close
                 puts "config::Successfully created telemetry file for prometheus sidecar"
               end
```

Comment:
did you test all these scenarios ? 1. No configmap applied 2. all these settings - https://github.com/microsoft/Docker-Provider/blob/ci_prod/build/windows/installer/conf/telegraf.conf#L133

Reply:
Tested scenario 1. 
For 2. I think is monitor_kubernetes_pods = false, then telegraf shouldn't be started right? Or is that not the case with Windows?  @ganga1980